dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/taglib/GlobalTagLibraryCache.java

```diff
@@ -582,23 +579,42 @@ private void loadTldFromClassloader(GlobalTagLibConfig globalTagLibConfig, TldPa
             try {
                 TagLibraryInfoImpl tli = tldParser.parseTLD(tldInputSource, is, globalTagLibConfig.getJarURL().toExternalForm());
                 if (tli.getReliableURN() != null) {
-                    if (containsKey(tli.getReliableURN()) == false) {
+                    
                         if (com.ibm.ejs.ras.TraceComponent.isAnyTracingEnabled() && logger.isLoggable(Level.FINE)) {
                             logger.logp(Level.FINE, CLASS_NAME, "loadTldFromClassloader", "Global jar tld loaded for {0}", tli.getReliableURN());
                         }
-                        tli.setURI(tli.getReliableURN());
-                        put(tli.getReliableURN(), tli);
-                        tldPathConfig.setUri(tli.getReliableURN());
```

Comment:
getReliableURN - https://docs.oracle.com/javaee/7/api/javax/servlet/jsp/tagext/TagLibraryInfo.html#getReliableURN--

Reply:
We set it here: https://github.com/OpenLiberty/open-liberty/blob/integration/dev/com.ibm.ws.jsp/src/com/ibm/ws/jsp/taglib/TldParser.java#L426

Reply:
`tldPathConfig.setUri(tli.getReliableURN());` override the URI set via the SPI.  I don't see where tldPathConfig is used again, but I'm getting it's for consistently since the URI within the TLD is the ones that's actually used.